uint8 buttons
int16 odom_buffer_x
int16 odom_buffer_y
int16 odom_buffer_th
uint8 line_sensors
uint16 gp_sensors
